src/poliastro/plotting.py

```diff
@@ -59,7 +59,7 @@ class OrbitPlotter(object):
 
     """
 
-    def __init__(self, ax=None, num_points=150):
+    def __init__(self, fig=None, ax=None, num_points=150):
```

Comment:
What if `ax` is not `None` but `fig` is? Perhaps instead of passing the figure, you can do

```
if ax:
    self.ax = ax
    self.fig = ax.figure
else:
    self.fig, self.ax = plt.subplots(...)
```